.pre-commit-config.yaml

```diff
@@ -21,3 +21,5 @@ repos:
     hooks:
       - id: mypy
         args: [--strict]
+        additional_dependencies: ['pytest']
```

Comment:
Is this change related to the other changes in the PR? (it's not clear)

Reply:
Oh right actually this broke a while ago, I ran into issues too and thought it may be a problem with me needing to re-create my env or something, but I never took the time to look at it more closely...
=> resolving